src/actions/index.js

```diff
@@ -42,13 +42,15 @@ export const selectCategories = (locationId, params) => (dispatch) => {
   });
 };
 
-export const updateLocation = (locationId, params) => (dispatch) => {
+export const updateLocation = (locationId, params, metaDataSection, fieldName) => (dispatch) => {
   // optimistically update the data store
   dispatch({
     type: OPTIMISTIC_UPDATE_LOCATION,
     payload: {
       id: locationId,
       params,
+      metaDataSection, 
```

Comment:
What do these new params (`metaDataSection` and `fieldName`) do?

Reply:
These are used to look up the last_updated_date field on the location metadata object. We pass them through to the reducer so that the reducer is also able to look up the metadata field in the location object in the store, and optimistically update the last_updated_date field in the store, so that the we can update the freshness labels in the UI after the field as been updated/confirmed.